File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    a = {}
    for i in range(len(clothes)):
        if clothes[i][1] in a:
            a[clothes[i][1]] += 1
        else:
            a[clothes[i][1]] = 1
            
    for i in a.values():
        answer *=  i + 1 
        
    return answer - 1

File: 프로그래머스/2/42578. 의상/의상.js
function solution(clothes) {
    var answer = 1;
    let a = {};
    for (let i = 0; i < clothes.length; i++) {
        let type = clothes[i][1];
        if (clothes[i][1] in a) {
            a[type]++
        } else {
            a[type] = 1
        }
    }
    
    for (let i in a) {
        answer *= (a[i] + 1)
    }
    
    return answer - 1;
}